data/src/main/java/io/gnosis/data/models/transaction/UnifiedEntry.kt

```diff
@@ -0,0 +1,46 @@
+package io.gnosis.data.models.transaction
+
+import com.squareup.moshi.Json
+import java.util.*
+import io.gnosis.data.models.transaction.Transaction as TransactionSummary
+
+enum class UnifiedEntryType {
```

Comment:
UnifiedEntryType sounds cool, but it's not specific enough. I think TxListEntryType would be more specific. Unified is a very generic term. Lots of things could be unified and we don't know what might be unified in the future.